internal/uvm/create.go

```diff
@@ -84,6 +84,10 @@ type Options struct {
 	// CPUGroupID set the ID of a CPUGroup on the host that the UVM should be added to on start.
 	// Defaults to an empty string which indicates the UVM should not be added to any CPUGroup.
 	CPUGroupID string
+
+	// ResourcePoolID holds the resource pool ID the UVM should be assigned to.
+	ResourcePoolID string
```

Comment:
can you make this `guid.GUID` and add an `"internal/oci".parseAnnotationsGUID(...) (guid.GUID, error)` function so we dont pass incorrect/unformatted GUIDs to HCS?

Reply:
It's a GUID in the mars schema, but it's serialized as a string in the JSON.